packages/suite/src/types/suite/index.ts

```diff
@@ -37,8 +37,16 @@ import type { MessageSystemAction } from '@suite-actions/messageSystemActions';
 import type { GuideAction } from '@suite-actions/guideActions';
 import type { Route } from '@suite-constants/routes';
 
-// reexport
-export type { ExtendedMessageDescriptor } from '@suite-components/Translation/components/BaseTranslation';
+// reexports
+import type {
+    ExtendedMessageDescriptor as IntlPackageExtendedMessageDescriptor,
+    MessageId,
+} from '@suite/messages';
+/**
+ * @deprecated Import `ExtendedMessageDescriptor` from `@suite/intl` instead.
```

Comment:
I don't want to change milion imports over whole app, at least not in same PR as this because it will cause lot of conflict that I will have to solve again and again during rebase. This could be done is some quicky merged PR after this one will be done.

Reply:
Yes!